src/multithreading.jl

```diff
@@ -0,0 +1,40 @@
+function task_local end
+
+"""
+    task_local(x::T) -> T
+
+Duplicate `x` for a new task such that it can be used concurrently with the original `x`.
+"""
+task_local(::Any)
+
+# Vector/Matrix (e.g. local matrix and vector)
+# task_local(A::Array) = copy(A)
```

Comment:
```suggestion
```
I just looked at this yesterday and wanted to suggest adding a definition like below 👍